e2e/utils.go

```diff
@@ -351,6 +351,28 @@ func writeDataInPod(app *v1.Pod, f *framework.Framework) error {
 	return err
 }
 
+//calculate md5sum
+func calMd5sum(app *v1.Pod, f *framework.Framework) (string, error) {
```

Comment:
@Yuggupta27 introduces `calculateSHA512sum()` for similar tests, please check with him and use a single checksum function

Reply:
Yes please, You can find it here https://github.com/ceph/ceph-csi/pull/1464/files#diff-c62628027491c2b96afe9b96aa706363R221